gamemodes/terrortown/gamemode/client/cl_hud_editor.lua

```diff
@@ -62,10 +64,15 @@ local function TryResizeLocalHUD(elem, trans_data, dif_x, dif_y, shift_pressed)
 	local additional_w, additional_h
 
 	if shift_pressed and trans_data.edge or elem:AspectRatioIsLocked() then
-		if dif_x * trans_data.direction_x * client.size.h > dif_y * trans_data.direction_y * client.size.w then
-			dif_x = mathRound(dif_y * trans_data.direction_y * client.aspect) * trans_data.direction_x
+		if
+			dif_x * trans_data.direction_x * client.size.h
+			> dif_y * trans_data.direction_y * client.size.w
+		then
+			dif_x = mathRound(dif_y * trans_data.direction_y * client.aspect)
+				* trans_data.direction_x
 		else
-			dif_y = mathRound(dif_x * trans_data.direction_x / client.aspect) * trans_data.direction_y
+			dif_y = mathRound(dif_x * trans_data.direction_x / client.aspect)
+				* trans_data.direction_y
```

Comment:
I think this shouldn't be done

Reply:
i'd rather not scroll horizontally

Reply:
![image](https://github.com/TTT-2/TTT2/assets/13639408/4f8615f0-8916-484c-a71d-43b0d37caaa4)
 but you don't have to? It is not like the line is 500 chars long. It is 103 chars long.

Reply:
that's 3 chars too long for github